extensions/reviewed/URLTools.json

```diff
@@ -109,7 +109,7 @@
           "disabled": false,
           "folded": false,
           "type": "BuiltinCommonInstructions::JsCode",
-          "inlineCode": "eventsFunctionContext.returnValue =\n    (new URL(eventsFunctionContext.getArgument(\"URL\")))[eventsFunctionContext.getArgument(attr)];\n",
+          "inlineCode": "eventsFunctionContext.returnValue =\n    (new URL(eventsFunctionContext.getArgument(\"URL\")))[eventsFunctionContext.getArgument(\"attr\")];\n",
```

Comment:
Should you increase the version number so that users are aware there's a new version?

Reply:
Oh yes you're right!